vllm/v1/engine/async_llm.py

```diff
@@ -138,6 +136,7 @@ def from_engine_args(
         engine_args: AsyncEngineArgs,
         start_engine_loop: bool = True,
         usage_context: UsageContext = UsageContext.ENGINE_CONTEXT,
+        stat_loggers: Optional[dict[str, StatLoggerBase]] = None,
```

Comment:
@liuzijing2014 could you also add something in the docstring here mentioning that the `stat_loggers` arg is unstable. I know it's documented on those data classes now but wouldn't harm to make it even clearer :)